### PR TITLE
Form Editor (Part 1)

### PR DESCRIPTION
**Pull request by SteveShengStar**
I am building a form editor (similar to Google Forms), that allows you to create custom forms. Please see Description and Figma Designs below to help you in reviewing this Pull Request.

Description:
[Form Editor Design Specs.docx](https://github.com/waterloop/teamhub/files/9126969/Form.Editor.Design.Specs.docx)

Figma Designs:
https://www.figma.com/file/mKZd1CsAcIdQcWRxlwyP4h/Term-Update-Form-Designs

### CODE REVIEW

File: backend/data/handlers/exportsheet.js [hunk 1]
```diff
@@ -23,14 +23,7 @@ exportsheet.readfile = async (token) => {
 
 exportsheet.writeTeamRoster = async (token) => {
     const fields = getFields(TEAM_ROSTER_FIELDS);
-    let result = await getAll(fields); // Get all the teamhub members' information
-
-    // console.log("databaseResult *******************************");
```
Comment (sashco-m) [inline backend/data/handlers/exportsheet.js:28]: nitpick but delete comments

File: frontend/components/organisms/formsection/Selectable.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+import { SystemComponent, SystemSpan } from '../../atoms/SystemComponents';
+import TextField from '@mui/material/TextField';
+import GhostButton from '../../atoms/GhostButton';
+
+const getModifiersBySectionType = (type) => {
+    switch(type) {
+        case "checkbox": 
+            return [{name: "multi", label: "Users can select multiple values", selected: true}]
+        case "radio": 
+            return [{name: "multi", label: "Users can select multiple values", selected: false}]
```
Comment (sashco-m) [inline frontend/components/organisms/formsection/Selectable.js:14]: could we do fallthrough here instead?

File: pages/api/google/export/roster.js [hunk 3]
```diff
@@ -6,10 +6,10 @@ module.exports = async (req, res) => {
     if (req.method === 'POST') {
         
         // Get the Access Token from the request headers
-        const token = cookie.parse(req.headers.cookie).token;
-        const authStatus = await data.auth.checkAnyUser(`Bearer ${token}`, res);
```
Comment (sashco-m) [inline pages/api/google/export/roster.js:10]: do we want to remove auth here

Comment (SteveShengStar) [reply pages/api/google/export/roster.js:10]: No we don't, I will open a Part 3 Merge Request soon to add this back in.

File: frontend/components/organisms/formsection/Selectable.js (hunk 2 above)
Comment (SteveShengStar) [reply frontend/components/organisms/formsection/Selectable.js:14]: Good point, will update.

**Pull request merged**